### PR TITLE
Add missing -D flag when dumping full systemd log

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

File: logexporter/cmd/main.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func createSystemdLogfile(service string, outputMode string, outputDir string) e
 
 // createFullSystemdLogfile creates logfile for full systemd journal in the outputDir.
 func createFullSystemdLogfile(outputDir string) error {
-	cmd := exec.Command("journalctl", "--output=short-precise")
+	cmd := exec.Command("journalctl", "--output=short-precise", "-D", "/var/log/journal")
```
Comment (fejta) [inline logexporter/cmd/main.go]: This adds a dependency on the internal details of where logexporter-daemonset.yaml mounts varlog, which seems brittle and hard to discover.

Can we make this a flag and update the logexporter-test container to configure this flag in its args (aka add a `--journal-path` similar to how we already have a `--gcs-path`.

Comment (mm4tt) [reply logexporter/cmd/main.go]: Good point, done. 
Needed to update one more place where the /var/log/journal was already hard-coded.

**Pull request merged**